ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/datacolumns/DataColumnSidecarCustodyImpl.java

```diff
@@ -110,26 +110,12 @@ public DataColumnSidecarCustodyImpl(
     this.blockResolver = blockResolver;
     this.nodeId = nodeId;
     this.totalCustodySubnetCount = totalCustodySubnetCount;
-    this.eip7594StartEpoch = spec.getForkSchedule().getFork(SpecMilestone.EIP7594).getEpoch();
+    this.minCustodyPeriodSlotCalculator = MinCustodyPeriodSlotCalculator.createFromSpec(spec);
```

Comment:
there is some risk of creating calculator two times separately here and there so it could be different calculators, I'd better create it once and pass same instance when creating DbAccessor and Custody

Reply:
Yes, you are right. Though it's a simple stateless class and no risk of duplicating its instances, but that could change in the future. So it's better to keep the things explicit.